src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -494,16 +499,15 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
   }
 
   /** Whether to show the editable forms for the addresses (true) or the static display addresses (false). */
-  private isEditing = false
+  protected isEditing = false
 
-  /** V-model for dropdown menu. */
-  protected dropdown = false
+  protected dropdown = false // v-model for dropdown menu
 
   // The 4 addresses that are the current state of the BaseAddress sub-components:
-  private mailingAddress: AddressIF = {}
-  private deliveryAddress: AddressIF = {}
-  private recMailingAddress: AddressIF = {}
-  private recDeliveryAddress: AddressIF = {}
+  protected mailingAddress = {} as AddressIF
+  protected deliveryAddress = {} as AddressIF
+  protected recMailingAddress = {} as AddressIF
+  protected recDeliveryAddress = {} as AddressIF
```

Comment:
I prefer this style since it's easier to see that all properties are initialized (which is required if you want them reactive -- undefined variables are not reactive).